### PR TITLE
Improve dashboard test

### PR DESCRIPTION
**Pull request by howardjohn**
Wait for only 20s instead of 100s, to increase test time

Stop sending HTTP traffic to TCP port

Send to all clusters in parallel. Before it was 100s * 5 clusters = 500s per test

### CODE REVIEW

**Comment by howardjohn** [comment]
cc @gargnupur

File: tests/integration/telemetry/dashboard_test.go [hunk 1]
```diff
@@ -339,15 +339,15 @@ func setupDashboardTest(t framework.TestContext) {
...
-		g, _ := errgroup.WithContext(context.Background())
-		tcpAddr := ing.TCPAddress()
-		ticker := time.NewTicker(time.Second * 5)
-		for t := 0; t < 20; t++ {
-			<-ticker.C
-			g.Go(func() error {
+	// Send 200 http requests, 20 tcp requests across goroutines, generating a variety of error codes.
+	// Spread out over 20s so rate() queries will behave correctly
+	g, _ := errgroup.WithContext(context.Background())
+	ticker := time.NewTicker(time.Second)
```
Comment (carolynhu) [inline tests/integration/telemetry/dashboard_test.go:345]: @howardjohn your ticker here should be times 20? according to your above comment,  `Spread out over 20s`

Comment (howardjohn) [reply tests/integration/telemetry/dashboard_test.go:345]: we have a loop above this. So 20 iterations, every 1s

Comment (carolynhu) [reply tests/integration/telemetry/dashboard_test.go:345]: got it! approval!

**Review by carolynhu — APPROVED** [review]
LGTM

**Review by gargnupur — APPROVED** [review]

**Pull request merged**